src/main/java/bio/terra/cli/command/group/Delete.java

```diff
@@ -19,8 +19,10 @@
   /** Delete an existing Terra group. */
   @Override
   protected void execute() {
-    deletePromptOption.confirmOrThrow();
     Group group = Group.get(groupNameOption.name);
+    String description =
+        String.format("This group contains %d members.", group.getMembers().size());
+    deletePromptOption.confirmOrThrow(description);
```

Comment:
This delete command uses a different pattern than the workspace/resource delete command. I think it would be good to be consistent i.e. include this in the description and print that before the prompt here too.

Reply:
good catch